addon/Config.jsm

```diff
@@ -1,58 +1,16 @@
 "use strict";
 
-/* to use:
+/* eslint no-unused-vars: ["error", { "varsIgnorePattern": "(Config|EXPORTED_SYMBOLS)" }]*/
 
-- Recall this file has chrome privileges
-- Cu.import in this file will work for any 'general firefox things' (Services,etc)
-  but NOT for addon-specific libs
-*/
+const EXPORTED_SYMBOLS = ["Config"];
 
-// Note: do not change "var" to "const" or Cu.import will end up with config undefined
-
-/* eslint no-unused-vars: ["error", { "varsIgnorePattern": "(config|EXPORTED_SYMBOLS)" }]*/
-var EXPORTED_SYMBOLS = ["config"];
-
-var config = {
-  "study": {
-    "studyName": "esper-pioneer-shield-study", // no spaces, for all the reasons
-    "forceVariation": {
-      "name": "pioneer",
-    }, // optional, use to override/decide
-    "weightedVariations": [
-      {"name": "pioneer",
-        "weight": 1},
-    ],
-    /** **endings**
-      * - keys indicate the 'endStudy' even that opens these.
-      * - urls should be static (data) or external, because they have to
-      *   survive uninstall
-      * - If there is no key for an endStudy reason, no url will open.
-      * - usually surveys, orientations, explanations
-      */
-    "endings": {
-      /** standard endings */
-      // NONE, this study has no surveys
-    },
-    "telemetry": {
-      "send": true, // assumed false. Actually send pings?
-      "removeTestingFlag": true,  // Marks pings as testing, set true for actual release
-      // TODO "onInvalid": "throw"  // invalid packet for schema?  throw||log
-    },
-    // relative to bootstrap.js in the xpi
-    "studyUtilsPath": `./StudyUtils.jsm`,
-  },
-  "isEligible": async function() {
-    // get whatever prefs, addons, telemetry, anything!
-    // Cu.import can see 'firefox things', but not package things.
-    return true;
-  },
-  "log": {
-    // Fatal: 70, Error: 60, Warn: 50, Info: 40, Config: 30, Debug: 20, Trace: 10, All: -1,
-    "bootstrap": {
-      "level": "Debug",
-    },
-    "studyUtils": {
-      "level": "Trace",
-    },
+const Config = {
+  addonId: "esper-pioneer-shield-study@shield.mozilla.org",
```

Comment:
this does not match the `addonId` in `package.json`.

Reply:
Thanks for spotting this! I'll rewrite the code to not even have to have an addonId in the config since it is more failsafe (against mistakes like this) to let the addon id be added at runtime.